g.mt

```diff
@@ -0,0 +1,38 @@
+abstract Interactable {
+    def interact(with: GameObject): void;
+}
+
+const OBJECT_POOL_SIZE: usize = 4096;
+
+enum PlayerKind {
+    Local
+    Remote
+    Unknown(String)
+
+    def is_known(self): bool {
+        return true;
+    }
+}
+
+def replace_with_zeroes(mut string: String): void {
+    // todo
+}
+
+import std.{fs.File, io.{open, close}};
+
+return;
```

Comment:
is this supposed to error?

Reply:
syntactically? no
semantically? yes, but that's not parser's job